### DISCUSSION TITLE
怎么批量下载课程付费课程？

### DISCUSSION THREAD

**Discussion by cxyax** [yutto-dev/yutto#295] [General]
首先感谢yutto的大佬🤩，工具真的很好用
我现在已经学会了单个下载付费课程，但是批量的命令我有点没看懂，**有没有交流群或者文档可以参考或咨询**
<img width="1075" alt="image" src="https://github.com/user-attachments/assets/a0ed9e8e-00f7-495f-9f0a-f3ef977b0681">

**Comment by lc4t** [comment]
readme有写，
```
批量参数
有些参数是只有批量下载时才可以使用的

点击展开详细参数
启用批量下载
参数 -b 或 --batch
默认值 False
只需要 yutto --batch <url> 即可启用批量下载功能。
```
 比如，`yutto -b https://www.bilibili.com/cheese/play/ss5918 -c 'xxxx'`

**Reply by cxyax** [reply]
感谢回复，但是我的课程每个链接地址不一样，参数 play/ 这里会变
![image](https://github.com/user-attachments/assets/fe407e1f-16b3-4e3d-aeb7-ed8f17102786)
![image](https://github.com/user-attachments/assets/513299de-cbd1-404e-8400-10f1920ce6b9)

**Reply by lc4t** [reply]
> 感谢回复，但是我的课程每个链接地址不一样，参数 play/ 这里会变 ![image](https://private-user-images.githubusercontent.com/58779674/358081774-fe407e1f-16b3-4e3d-aeb7-ed8f17102786.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjM2ODk4MDksIm5iZiI6MTcyMzY4OTUwOSwicGF0aCI6Ii81ODc3OTY3NC8zNTgwODE3NzQtZmU0MDdlMWYtMTZiMy00ZTNkLWFlYjctZWQ4ZjE3MTAyNzg2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODE1VDAyMzgyOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWVkZjQ5NjU3MDMxYzRiYTkwYWRhY2JiZTE3ZTEzYjQyZjQxNmYxODI3MWFlODZmZmE3ZTZhOTM3YTc4ZTY0MGQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.H02I32_e58a12KIAWC1lUoVRojhWt3uvARQut3QnMCk) ![image](https://private-user-images.githubusercontent.com/58779674/358081923-513299de-cbd1-404e-8400-10f1920ce6b9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjM2ODk4MDksIm5iZiI6MTcyMzY4OTUwOSwicGF0aCI6Ii81ODc3OTY3NC8zNTgwODE5MjMtNTEzMjk5ZGUtY2JkMS00MDRlLTg0MDAtMTBmMTkyMGNlNmI5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODE1VDAyMzgyOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWZmNDQ5ZWIxMDZmOTI1YjY5ZWEzYTdiNzBlNzE1MDVlZDllYjEzZjdhOTYyODFkMWVkZThiYTg4Y2FlYzUyOGYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.EggmWh8v_JIb2yIjiuExl7D3ucQMXkc0Cy8RYwsj_jY)

课程主页地址...不是单集地址...
<img width="524" alt="image" src="https://github.com/user-attachments/assets/ee9c3e94-1ef9-4bc6-92fe-5e2f33408d2b">

**Reply by cxyax** [reply]
可以了，感谢老哥解答👍
<img width="1260" alt="image" src="https://github.com/user-attachments/assets/7ab511dd-2726-42b4-b95f-8b50f0d84c63">
